javascript/photo.js

```diff
@@ -115,64 +115,65 @@ backgrounds.Photo = new Model({
         return imageData;
     },
 
+    isValidLocalStorageState: function(cacheSize) {
+        var count = localStorage.getItem("count");
+        var order = localStorage.getItem("order");
+        var cachedImages = localStorage.getItem("cachedImages");
+        return count != null && Number(count) >= 0 && order != null && JSON.parse(order).length == this.BACKUP_IMAGES_COUNT && cachedImages != null && JSON.parse(cachedImages).length <= cacheSize;
```

Comment:
Is count >= 0 required? Just a sanity check? Similar comments about the other checks.
Also, if possible split this return into three boolean checks for readability if possible.